tigerpath/majors_and_certificates/majors/ARC_2018.json

```diff
@@ -0,0 +1,194 @@
+    "https://soa.princeton.edu/sites/default/files/student%20handbook_undergrad%20F2019-DC.pdf"
+  ],
+  "contacts": [
+    {
+      "type": "Dean",
+      "name": "Mónica Ponce de León",
+      "email": "mpdl@princeton.edu"
+    },
+    {
+      "type": "Associate Dean",
+      "name": "Paul Lewis",
+      "email": "paul@ltlarchitects.com"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Mario I. Gandelsonas",
+      "email": "mgndlsns@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Cameron Wu",
+      "email": "cwu@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "Students who wish to concentrate in architecture are required to complete two courses during their first year or sophomore year: ARC 203 Introduction to Architectural Thinking and ARC 204 Introduction to Architectural Design. The courses do not need to be taken in sequence. At least one course in architectural history, taken in either the School of Architecture or the Department of Art and Archaeology, is recommended but not required to be completed before their junior year.",
+      "completed_by_semester": 4,
+      "course_list": [
+        "ARC 203",
+        "ARC 204"
+      ]
+    },
+    {
+      "name": "Departmentals",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "In addition to the general prerequisites and the requirements for independent work, each student is required to complete ten courses in three related areas: history and theory, technology, and design. The History and Theory distribution requires six courses: three History and Theory of Architecture courses, two of which are ARC 308 and ARC 403; two courses in History and Theory of Urbanism and Landscape; and at least two upper level courses to be taken in the Department of Art and Archaeology. The Technology distribution requires two courses, one of which is ARC 311: Building Science and Technology–Building Systems. The Design Seminar distribution requires ARC 206.\n\nArchitecture concentrators take the following sequence of courses in their junior and senior year: Junior Studio, which consists of ARC 350 and ARC 351, and two required history and theory courses: ARC 308: History of Architectural Theory (taken in the fall of the junior year) and ARC 403: Topics in the History and Theory of Architecture in the fall semester of the senior year. These courses introduce methodologies of historical analysis and research, the literature of the field, and varieties of architectural writing.\n\nIn the fall semester of their senior year, students are required to enroll in ARC 404: Advanced Design Studio. ARC 404 is centered around independent design projects in that synthesize students’ training and interests and investigate new approaches to representation.",

```

Comment:
I forgot about this, but this also hasn't been fixed yet. It should say **seven** courses, based on the new requirements.
```suggestion
      "explanation": "In addition to the general prerequisites and the requirements for independent work, each student is required to complete ten courses in three related areas: history and theory, technology, and design. The History and Theory distribution requires seven courses: three History and Theory of Architecture courses, two of which are ARC 308 and ARC 403; two courses in History and Theory of Urbanism and Landscape; and at least two upper level courses to be taken in the Department of Art and Archaeology. The Technology distribution requires two courses, one of which is ARC 311: Building Science and Technology–Building Systems. The Design Seminar distribution requires ARC 206.\n\nArchitecture concentrators take the following sequence of courses in their junior and senior year: Junior Studio, which consists of ARC 350 and ARC 351, and two required history and theory courses: ARC 308: History of Architectural Theory (taken in the fall of the junior year) and ARC 403: Topics in the History and Theory of Architecture in the fall semester of the senior year. These courses introduce methodologies of historical analysis and research, the literature of the field, and varieties of architectural writing.\n\nIn the fall semester of their senior year, students are required to enroll in ARC 404: Advanced Design Studio. ARC 404 is centered around independent design projects in that synthesize students’ training and interests and investigate new approaches to representation.",
```